<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>RegistrationBasedCorrespondence</class>
  <widget class="qMRMLWidget">
    <layout class="QVBoxLayout">
      <item>
        <widget class="ctkCollapsibleButton">
          <property name="text">
            <string>General I/O</string>
          </property>
          <layout class="QFormLayout">
            <item row="0" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Template Mesh</string>
                </property>
              </widget>
            </item>
            <item row="0" column="1">
              <widget class="ctkPathLineEdit" name="TemplateMesh">
                <property name="toolTip">
                  <string>Template Mesh</string>
                </property>
              </widget>
            </item>

            <item row="1" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Input Directory</string>
                </property>
              </widget>
            </item>
            <item row="1" column="1">
              <widget class="ctkDirectoryButton" name="InputDirectory">
                <property name="toolTip">
                  <string>Input Directory</string>
                </property>
              </widget>
            </item>

            <item row="2" column="0">
              <widget class="QLabel">
                <property name="text">
                  <string>Output Directory</string>
                </property>
              </widget>
            </item>
            <item row="2" column="1">
              <widget class="ctkDirectoryButton" name="OutputDirectory">
                <property name="toolTip">
                  <string>Output Directory</string>
                </property>
              </widget>
            </item>
          </layout>
        </widget>
      </item>

      <item>
        <widget class="ctkCollapsibleButton">
            <property name="text">
              <string>Advanced</string>
            </property>
            <property name="collapsed">
              <bool>true</bool>
            </property>
            <layout class="QFormLayout">
              <item row="0" column="0">
                <widget class="QRadioButton" name="DiffeoButton">
                  <property name="text">
                    <string>Diffeomorphic</string>
                  </property>
                  <property name="checked">
                    <bool>true</bool>
                  </property>
                </widget>
              </item>
              <item row="0" column="1">
                <widget class="QRadioButton" name="BSplineButton">
                  <property name="text">
                    <string>B-spline</string>
                  </property>
                  <property name="checked">
                    <bool>false</bool>
                  </property>
                </widget>
              </item>
              <item row="1" column="0">
                <widget class="QLabel">
                  <property name="text">
                    <string>Iterations</string>
                  </property>
                </widget>
              </item>
              <item row="1" column="1">
                <widget class="ctkSliderWidget" name="IterationsSlider">
                  <property name="decimals">
                    <number>0</number>
                  </property>
                  <property name="minimum">
                    <number>1</number>
                  </property>
                  <property name="maximum">
                    <number>5000</number>
                  </property>
                  <property name="value">
                    <number>200</number>
                  </property>
                </widget>
              </item>
          </layout>
        </widget>
      </item>

      <item>
        <widget class="QPushButton" name="ApplyButton">
          <property name="text">
            <string>Apply</string>
          </property>
        </widget>
      </item>

      <item row="100" column="0">
        <spacer name="verticalSpacer">
          <property name="orientation">
            <enum>Qt::Vertical</enum>
          </property>
        </spacer>
      </item>
    </layout>
  </widget>
  <customwidgets>
    <customwidget>
      <class>ctkCollapsibleButton</class>
      <extends>QWidget</extends>
      <header>ctkCollapsibleButton.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>qMRMLWidget</class>
      <extends>QWidget</extends>
      <header>qMRMLWidget.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>ctkSliderWidget</class>
      <extends>QWidget</extends>
      <header>ctkSliderWidget.h</header>
      <container>0</container>
    </customwidget>
  </customwidgets>
  <resources/>
  <connections/>
</ui>
